packages/components/nodes/vectorstores/Chroma/Chroma_Existing.ts

```diff
@@ -86,13 +93,18 @@ class Chroma_Existing_VectorStores implements INode {
         const credentialData = await getCredentialData(nodeData.credential ?? '', options)
         const chromaApiKey = getCredentialParam('chromaApiKey', credentialData, nodeData)
 
+        const chromaMetadataFilter = nodeData.inputs?.chromaMetadataFilter;
+        const metadataFilter = chromaMetadataFilter ? JSON.parse(chromaMetadataFilter) : {};
```

Comment:
Recommend following other `Metadata Filter` standard

Examples:
1. [packages/components/nodes/vectorstores/Pinecone_Existing/Pinecone_Existing.ts](https://github.com/FlowiseAI/Flowise/blob/main/packages/components/nodes/vectorstores/Pinecone_Existing/Pinecone_Existing.ts#L111)
2. [packages/components/nodes/vectorstores/Supabase_Existing/Supabase_Exisiting.ts](https://github.com/FlowiseAI/Flowise/blob/main/packages/components/nodes/vectorstores/Supabase_Existing/Supabase_Exisiting.ts#L108)

Reply:
@chungyau97 thx for reviewing it. 

I compared it to the pinecone_existing.ts and it look exactly to how I implemented chroma_existing.ts. Even the lines you highlighted above, I checked them and I couldn't find the difference. 

P.S. This is my second time using TS and my first PR for an open-source repo 🙂

Reply:
Hey @pooriaarab, sorry for the confusing lines shown above. 
No worries, I will guide you.

Line: 103
Should be `filter?: object | undefined` instead of `metadataFilter?: any`
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/ef1357d1-7516-4825-be5d-fa3f33a8c10a)

Lines: 97 &  107 should be combined and follow below format
```
        if (chromaMetadataFilter) {
            const metadatafilter = typeof chromaMetadataFilter === 'object' ? chromaMetadataFilter : JSON.parse(chromaMetadataFilter)
            obj.filter = metadatafilter
        }
```
Lastly, remember to run `yarn lint-fix` and share your results after implementation.

Reply:
Thx for the tips Yau!!!

Worked like a charm. Learned about git clone, add...

Only trouble I had was I cloned the main flowise repo but when I tried pushing it to the repo using "git push origin main" i got an error (i think it's because I don't have push access)

So I just ended up copy pasting it visually on the github website.

How can I push changes from my CLI?